### PR TITLE
card layout and create card functionality

### PR DESCRIPTION
**Pull request by dev-tang**

### CODE REVIEW

File: my-app/src/components/General/TaskCard.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import Button from 'react-bootstrap/Button';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import Container from 'react-bootstrap/Container';
+
+export default ({ title, taskGoal, description }) => {
+	const now = 85;
+
+	const progressInstance = <ProgressBar className="progressBar" animated now={now} label={`${now}min`} />;
+
+	const [card, setCard] = useState({
```
Comment (upluke) [inline my-app/src/components/General/TaskCard.js:18]: Task card should be a stateless component

**Comment by upluke** [comment]
Good work! I like the scroll bar haha! But it would be more desirable if:
1. TaskCard component should go stateless (plz see the comment)
2. Move the CreateTaskDialog component down below the cards section, so the section will not toggle around when create button is clicked 
3. Description display
4. Remove redundant/repeated components, like Card.js, GearMenu.js,etc.

File: my-app/src/components/General/TaskCard.js (hunk 1 above)
Comment (dev-tang) [reply my-app/src/components/General/TaskCard.js:18]: oh okay thank you, got it!

**Comment by dev-tang** [comment]
> Good work! I like the scroll bar haha! But it would be more desirable if:
> 
> 1. TaskCard component should go stateless (plz see the comment)
> 2. Move the CreateTaskDialog component down below the cards section, so the section will not toggle around when create button is clicked
> 3. Description display
> 4. Remove redundant/repeated components, like Card.js, GearMenu.js,etc.

alright, I'll make the TaskCard component stateless. got it, I'll move the CreateTaskDialog down below the cards section. Hmm the description wasn't displayed? I thought it was, I'll look into it right now - LoL I forgot about Card.js, I'll remove redundant components thanks!

**Pull request closed without merging**